backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/internalbusinesslogic/InvoiceCandidate.java

```diff
@@ -203,8 +222,7 @@ public ToInvoiceData computeToInvoiceData()
 					logger.debug("qtyDeliveredInUom={} is > qtyToInvoiceInUom={}; -> going to use qtyDelivered instead of override, for the UOM-qty",
 								 qtyDelivered.toBigDecimal(), qtysToInvoice.getUOMQtyNotNull().toBigDecimal());
 
-					final StockQtyAndUOMQty qtysToIvoiceWithAdjustedQty = qtysToInvoice.toBuilder().uomQty(qtyDelivered).build();
-					qtysEffective = qtysToIvoiceWithAdjustedQty;
+					qtysEffective = qtysToInvoice.toBuilder().uomQty(qtyDelivered).build();
```

Comment:
Shouldn't we differ between catchweight and nominal here?

Reply:
the check also done on de.metas.invoicecandidate.internalbusinesslogic.InvoiceCandidate#getQtysToInvoice